packages/elements/src/list/helpers/renderer.ts

```diff
@@ -12,47 +14,57 @@ type Context = {
   multiple?: boolean;
 };
 
+export const createListRenderer = <T extends DataItem = ItemData>(
+  context?: unknown
+): ((item: T, composer: CollectionComposer<T>, element?: HTMLElement) => HTMLElement) => {
```

Comment:
This looks unnecessary.
```suggestion
) => {
```